packages/applicant/src/pages/api/mandatory-questions/[mandatoryQuestionId]/create-submission.test.tsx

```diff
@@ -0,0 +1,143 @@
+import { merge } from 'lodash';
+import {
+  AdvertDto,
+  getAdvertBySchemeId,
+} from '../../../../services/GrantAdvertService';
+import {
+  GrantMandatoryQuestionDto,
+  GrantMandatoryQuestionService,
+} from '../../../../services/GrantMandatoryQuestionService';
+import {
+  CreateSubmissionResponse,
+  createSubmission,
+} from '../../../../services/SubmissionService';
+import { getJwtFromCookies } from '../../../../utils/jwt';
+import { routes } from '../../../../utils/routes';
+import handler from './create-submission.page';
+
+jest.mock('../../../../services/GrantAdvertService');
+jest.mock('../../../../services/SubmissionService');
+jest.mock('../../../../services/GrantMandatoryQuestionService.ts');
+jest.mock('../../../../utils/jwt');
+
+const mockedRedirect = jest.fn();
+const mockedSetHeader = jest.fn();
+const mockedSend = jest.fn();
+
+const req = (overrides: any = {}) =>
```

Comment:
Same for this one (if you plan to use it)

`{ [key: string]: Overrides | string };`